### DISCUSSION TITLE
Voces conhecem ou já trabalharam com Wolfram Language Image Identification Project?

### DISCUSSION THREAD

**Discussion by AmandaFonseca** [frontendbr/forum#651] [Perguntas e Respostas]
Oi pessoal andei pesquisando sobre reconhecimento de imagens descobrir essa solução da Wolfram Alpha porém não achei um material de estudo bom, somente artigos.

Alguém aqui já usou esse cara?

**Comment by rafa-acioly** [comment]
você já [se cadastrou](https://account.wolfram.com/auth/create) para ver a plataforma com a linguagem?

**Comment by AmandaFonseca** [comment]
@rafa-acioly sim eles tem ate uma especie de IDE in cloud o problema eh que o Wolfram Languagem tem uma documentação extensa que fala do uso da linguagem para aprender conceitos de matemática, editar imagens e mapas mas sobre como aplicar o reconhecimento de imagens num  projeto web eu não achei nada. Talvez essa resposta esteja embaixo do meu nariz mas como eu não sei Inglês não acho.  
Li o PDF deles e não achei o que eu preciso seria ótimo aprender todas as possibilidades da linguagem mas por enquanto isso sai da minha trilha de estudos pois vai levar muito tempo.
